addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -44,12 +44,7 @@
                             <span class="swissqr_text content" t-field="o.company_id.city"/><br/>
 
                             <span class="swissqr_text title">Reference</span><br/>
```

Comment:
Depending on your answer to my other comment, maybe we could indeed do a condition on l10n_ch_isr_valid and company.invoice_reference_type ==  'ch_isr', then just get the reference field.

Reply:
I agree, maybe this code is more suited for v13 as the `reference_type` disapeared. Sorry I forgot that we had it in v12.

Reply:
For the field name using `l10n_ch_isr_number_spaced` seems safer to me otherwise we need a new check that `reference` really contains a valid ISR reference in such case. (I have such proposal but only for v13). `Reference` on the QR report is only intended for structured reference (QRR or SCOR)


> **Reference** QR reference or Creditor Reference (ISO 11649). The QR reference is printed in blocks of 5 characters (beginning with 2 characters, then 5x5-character groups). The Creditor Reference is printed in blocks of 4 characters, the last block being able to contain less than 4 characters). 

(SCOR aka Creditor reference is not implemented yet and we should focus on QRR right now but we can keep in mind that can become an alternative in the future.)